kafka/arena-acl-ingestor/src/main/kotlin/no/nav/amt/tiltak/ingestors/arena_acl_ingestor/processor/DeltakerProcessor.kt

```diff
@@ -107,6 +107,8 @@ class DeltakerProcessor(
 	private fun ingestGjennomforing(gjennomforingId: UUID): Gjennomforing {
```

Comment:
Bare en tanke som slo meg når jeg leste dette og litt urelatert til oppgaven: Vi bruker ofte ordet `ingest` for å mene at vi leser inn en melding fra kafka så jeg tenkte først at dette var i GjennomføringsIngestoren. Jeg tror det ville vært litt enklere å forstå om det var bare `upsertGjennomforing()`.

Reply:
Ja, enig! Jeg ble forvirret av dette selv da jeg jobbet med denne delen av koden :)  Skal endre.